doc/source/whatsnew/v0.16.0.txt

```diff
@@ -546,6 +546,37 @@ Bug Fixes
 - Bug in ``rank`` where comparing floats with tolerance will cause inconsistent behaviour (:issue:`8365`).
 - Fixed character encoding bug in ``read_stata`` and ``StataReader`` when loading data from a URL (:issue:`9231`).
 - Bug in adding ``offsets.Nano`` to other offets raises ``TypeError`` (:issue:`9284`)
+<<<<<<< HEAD
+=======
+
+- Fixed bug (:issue:`9542`) where labels did not appear in legend (instead, ``None`` types were displayed for each series) when passing ``label=`` args to ``dataframe.plot()``. Also, series indices are no longer mutated and labels are default to column names if ``label=`` args are not provided. Use ``legend=False`` to avoid printing a legend.
+
+  Fixed behavior:
```

Comment:
cool i added this to the issue (not sure how to bump it to the top). Thanks @TomAugspurger